src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -444,32 +452,29 @@ namespace UtilityRoutines {
     // The finalColumn (an optional argument) being true triggers the actual file to be written or appended.
     // J.Glazer February 2020
     {
-        static std::string headerRow = "";
-        static std::string valuesRow = "";
-
         // the following was added for unit testing to clear the static strings
         if (colHeader == "RESET" && colValue == "RESET") {
-            headerRow = "";
-            valuesRow = "";
+            appendPerfLog_headerRow = "";
```

Comment:
I think a more ideal thing would be to just have a function called `clearPerfLogHeaders()` rather than relying on special header names to clear these.  But that is OK for right now.  Next time you work on this area, please clean this out.